RELEASES.md

```diff
@@ -1,8 +1,22 @@
-# Release Process
-This document explains the process to release new version and to address patch release.
+# Releases
+This document explains the process to release new versions.
+
+## Versioning
+Versions are expressed as `vX.Y.Z` as well as Semantic Versioning;
+
+- `X` is the major version
+- `Y` is the minor version
+- `Z` is the patch version
+
+## Release Cycle
+- Major releases: arbitrary timing
+- Minor releases: roughly every 2 months
+- Patch releases: roughly every 2-3 weeks
+
+Release candidates(vX.X.X-alpha) for minor versions are available for 2-4 weeks before the release.
 
 ## Major release
-This refers to the release of new features.
+This refers to the release of breaking changes.
```

Comment:
I don't think we're following this right? 🤔 I mean, we have not released any major yet, and minor does contain breaking changes. Directly mentioning that we're not follow the Semantic version maybe better.